project.clj

```diff
@@ -18,14 +18,15 @@
                  [puppetlabs/kitchensink ~ks-version]
                  [puppetlabs/certificate-authority "0.6.0"]
                  [puppetlabs/http-client "0.4.0"]
-                 [org.jruby/jruby-core "1.7.15" :exclusions [com.github.jnr/jffi com.github.jnr/jnr-x86asm]]
+                 [org.jruby/jruby-core "1.7.19"
+                  :exclusions [com.github.jnr/jffi com.github.jnr/jnr-x86asm]]
+                 [com.github.jnr/jffi "1.2.8"]
```

Comment:
The whole reason I had to add these in the first place was because lein complains about them specifying a dependency as a range, e.g. "[1.2.1,1.3.0)" or something, which would lead me to believe that they were getting 1.2.8 in their builds, but I haven't done anything via maven.